src/forms/form-control.scss

```diff
@@ -70,11 +70,7 @@ textarea.form-control {
 
 // Inputs with contrast for easy light gray backgrounds against white.
 .input-contrast {
-  background-color: var(--color-canvas-inset);
-
-  &:focus {
-    background-color: var(--color-canvas-default);
-  }
+  background-color: var(--color-canvas-default);
```

Comment:
It looks like this change renders `.input-contrast` to be the same as the default `.form-control` styles, is that right? Not totally sure the point of this class. I wonder if it makes more sense to deprecate it and use default

Reply:
ah, indeed. for my fix maybe it's better just to remove this class and that audit can be done independently. thank you @langermank